django/db/models/expressions.py

```diff
@@ -522,6 +526,24 @@ def copy(self):
         copy.extra = self.extra.copy()
         return copy
 
+    @property
+    def lhs(self):
+        """
+        This property exists for compatibility with the previously
```

Comment:
I guess we can keep it. It doesn't cause any serious maintenance overhead (as long as it is tested).